.github/workflows/bridge-release.yml

```diff
@@ -1,11 +1,61 @@
-name: Webhook Bridge Release
```

Comment:
Nitpick: I'd maybe call the directory `svix-bridge`, so it feels like it's a separate binary rather than a small chunk of the server.

Reply:
Can do, though the `bridge/` name has an affinity with `server/` in that:
- `server/svix-server`
- `bridge/svix-bridge`

Fine by me either way. LMK what you want and I can find/replace my way to it.